tests/test.ts

```diff
@@ -1,13 +1,13 @@
 import { assert, expect } from 'chai';
-import type Club from '../models/club';
-import DbProvider from '../backend_tools/db_provider';
+import type Club from '@src/models/club';
```

Comment:
I may've missed it, though where is `@src` defined? I assume it defaults to the `src` folder?

Reply:
`@src` is defined in the tsconfig.json. It's just a way to make certain file paths shorthand

Reply:
I must've not seen it there! I checked in the project config and didn't see it, but looks like you're right